src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4620,9 +4617,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
 -warnaserror[+|-]             Report all warnings as errors
 -warnaserror[+|-]:&lt;warn list&gt; Report specific warnings as errors
                               (use "nullable" for all nullability warnings)
--warnversion:&lt;n&gt;        Enable recently introduced warnings. Default '0'.
-                              To enable warnings introduced through this version, use '5'.
-                              Use '9999' to enable all current and future warning waves.
+-warn:&lt;n&gt;                     Set warning level (0 or higher) (Short form: -w)
```

Comment:
It feels like much of the help text here for '-warnversion' is now applicable to '-warn'. Why remove it?

Reply:
Tracked at [https://github.com/dotnet/roslyn/issues/46218](https://github.com/dotnet/roslyn/issues/46218)

---
In reply to: [458391431](https://github.com/dotnet/roslyn/pull/46193#discussion_r458391431) [](ancestors = 458391431)